docs/03-explore.md

```diff
@@ -39,11 +39,16 @@ launched, you can execute the following:
 
     oc logs deployments/cluster-version-operator -n openshift-cluster-version > operatorlog.txt
 
-The operator's log is **extremely** long, so it is recommended that you redirect it to a file instead of trying to look at it directly with the `logs` command.
+The operator's log is **extremely** long, so it is recommended that you
+redirect it to a file instead of trying to look at it directly with the
+`logs` command.
 
-# Exploring RHEL CoreOS
-The OpenShift 4 cluster is made of hosts that are running RHEL CoreOS.
-CoreOS is a container optimized, hardened, minimal footprint
+# Exploring Red Hat CoreOS
```

Comment:
It should be RHEL CoreOS, according to http://post-office.corp.redhat.com/archives/aos-devel/2019-February/msg00113.html

Reply:
Ironically the MOTD when you SSH in I think still says Red Hat CoreOS. However this text is fixed.